### PR TITLE
Implement isDeviceSupportedAsResizeTarget

### PR DESCRIPTION
**Pull request by AndreyBelym**
\cc @inikulin @helen-dikareva @AlexanderMoskovkin

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -12,5 +13,6 @@ export default {
     open,
     close,
     resize,
-    screenshot
+    screenshot,
+    isDeviceSupportedAsResizeTarget
```
Comment (AlexanderMoskovkin) [inline src/index.js]: I suppose we should add the new api method to the API documentation?

File: src/api/resize/utils.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    var size = null;
+
+    while (devices.length && !size)
+        size = parseSize(devices.shift().size);
+
+    return size;
+}
+
+export async function isDeviceSupported (device) {
+    var size = await getDeviceSize(device);
```
Comment (inikulin) [inline src/api/resize/utils.js]: Oops, we can't use async functions in validations. We need to figure out other way to do this

Comment (inikulin) [inline src/api/resize/utils.js]: As an option we can use late errors and validate device name in `executeWindowTaskHandler` and just validate for non-empty string on early execution

Comment (helen-dikareva) [inline src/api/resize/utils.js]: Ok, let's use `isDeviceSupported` method before doing resize

**Comment by AndreyBelym** [comment]
Closing this in favor for #63.

**Pull request closed without merging**